wgetpaste

```diff
@@ -613,7 +613,7 @@ showservices() {
 		((INDV=3+max+IND))
 		engine=" $E[${INDV}G| Pastebin engine:"
```

Comment:
You could probably remove the colon here too. Running `-S --verbose` adds an extra column.

Reply:
ack. done